### PR TITLE
Added New Function +semver: minor

### PR DESCRIPTION
**Pull request by JCarnall**

### CODE REVIEW

File: DemoApp/Program.cs [hunk 1]
```diff
@@ -1,77 +1,8 @@
...
-using System.Data.SqlClient;
-using Dapper;
+﻿using System;
+using DemoApp;
 
-var sqlDataRepository = new SqlDataRepository();
-Console.WriteLine((await sqlDataRepository.Perils)["ALL"]);
-Console.WriteLine((await sqlDataRepository.StatTypes)["AAL"]);
-Console.ReadLine();
+var primeService = new PrimeService();
```
Comment (asorrentino84) [inline DemoApp/Program.cs:5]: Jon you are too precise.

Comment (asorrentino84) [reply DemoApp/Program.cs:5]: @JCarnall really like the prime numbers!

Comment (JCarnall) [reply DemoApp/Program.cs:5]: yes I do, we need more primes

Comment (JCarnall) [reply DemoApp/Program.cs:5]: @asorrentino84

Comment (asorrentino84) [reply DemoApp/Program.cs:5]: good!

**Review by asorrentino84 — APPROVED** [review]
Looks good!

**Pull request merged**